pkgs/desktops/gnome/artwork/firewatch/default.nix

```diff
@@ -0,0 +1,230 @@
+{ stdenv
+, lib
+, fetchzip
+, gnome
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wallpaper-firewatch";
+<<<<<<< HEAD
+  version = "2.1";
+=======
+  version = "0.1";
+>>>>>>> a11990b425c31bde2882fa77d926db610b5b563f
+  src = fetchzip {
+    url = "https://cdn.manishk.dev/Firewatch.zip";
```

Comment:
Is this URL guaranteed not to change? Otherwise, it makes a bad fixed-output derivation.

Reply:
The upstream repo doesn't include the resources.
[install.sh](https://github.com/manishprivet/dynamic-gnome-wallpapers/blob/c953f3d876187b910eb540f6a457465c6788245b/install.sh#L63) uses string substitution to download depending on "download code", which is just the filename.   
If the following is undesired, I can mirror said resources and guarantee they will never change